documentation/modules/ROOT/partials/modules/all-connectors/con-connector-ad-hoc-snapshots.adoc

```diff
@@ -0,0 +1,93 @@
+ifdef::community[]
```

Comment:
Is this present in MongoDB too? It should starting 1.8

Reply:
I've conditionalized the include statements and the related heading to filter them from the downstream content for now. We can remove the conditions in a future release.